### PR TITLE
[docs] Cloud docs (#9920)

### PR DESCRIPTION
**Pull request by ddhodge**
Jira 2092 2094 2097 2098 2099
as well as Password reset instructions

@netlify /latest/yugabyte-cloud

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-clusters/overview.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The following **Key Metrics** are tracked on the **Overview** tab:
 - Operations/sec - Shows the read and write operations on the cluster over time.
 - Average Latency - Shows the average amount of time in milliseconds taken for read and write operations.
 - CPU Usage - Shows the percentage of CPU use for the cluster.
-- Disk Usage - Shows the amount of disk space used by the cluster.
+- Disk Usage - Shows the amount of disk space provisioned for and used by the cluster.
 
 Metrics can be displayed over the past hour, 6 hours, 12 hours, 24 hours, or 7 days.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-clusters/overview.md:27]:
```suggestion
You can show metrics for the past hour, 6 hours, 12 hours, 24 hours, or 7 days.
```

File: docs/content/latest/yugabyte-cloud/cloud-faq.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ showAsideToc: true
 
 Yugabyte Cloud is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on Google Cloud Platform (GCP), Amazon Web Services (AWS), and Microsoft Azure (coming soon).
 
-Yugabyte Cloud clusters are accessed via [YSQL](../../api/ysql) and [YCQL](../../api/ysql) client APIs, and administered using the Yugabyte Cloud console.
+Yugabyte Cloud clusters are accessed via [YSQL](../../api/ysql) and [YCQL](../../api/ycql) client APIs, and administered using the Yugabyte Cloud console.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-faq.md:22]:
```suggestion
You access your Yugabyte Cloud clusters via [YSQL](../../api/ysql) and [YCQL](../../api/ycql) client APIs, and administer your clusters using the Yugabyte Cloud console.
```

File: docs/content/latest/yugabyte-cloud/cloud-quickstart.md [hunk 3]
```diff
@@ -46,7 +46,9 @@ Once the Sign up process completes, the **Getting Started** page of the Yugabyte
 
 1. Choose the provider (AWS or GCP), enter a name for the cluster, and choose the region, then click **Next**.
 
-1. Click **Copy credentials** and save the credentials in a secure location. This creates a database user named "admin" that you will use to connect to your YugabyteDB database in the cloud. The database name is "yugabyte".
+1. Choose the credentials you will use to connect to your YugabyteDB database in the cloud. You can choose the default set with a database user named "admin", or create your own.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart.md:49]:
```suggestion
1. Choose the credentials you'll use to connect to your YugabyteDB database in the cloud. You can choose the default set with a database user named "admin", or create your own.
```

File: docs/content/latest/yugabyte-cloud/cloud-quickstart.md [hunk 4]
```diff
@@ -46,7 +46,9 @@ Once the Sign up process completes, the **Getting Started** page of the Yugabyte
...
+
+1. Click **Copy credentials** save your credentials in a secure location.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart.md:51]:
```suggestion
1. Click **Copy credentials** and save your credentials in a secure location.
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
Quick scrub over everything... One comment: careful with consistency in list items and run-in heads. Sometimes you use colons, sometimes not.

File: docs/content/latest/yugabyte-cloud/cloud-basics/add-connections.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Yugabyte Cloud only allows client connections to clusters from entries in IP allow lists that have been explicitly assigned to the cluster. Each entry in the lists is either a single IP address or a CIDR-notated range of addresses.
+Yugabyte Cloud only allows client connections to clusters from entries in IP allow lists that have been explicitly assigned to the cluster. Each entry can either be a single IP address, a CIDR-notated range of addresses, or multiple comma-delimited addresses.
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-basics/add-connections.md:16]: Allow lists can be assigned to and shared among any cluster, and clusters can have any number of allow lists.

File: docs/content/latest/yugabyte-cloud/cloud-basics/connect-to-clusters.md [hunk 6]
```diff
@@ -155,25 +155,32 @@ sslrootcert=<ROOT_CERT_PATH>
...
 
 ```sh
-cassandra://<DB USER>:<DB PASSWORD>@e53ea9b6-424a-4db5-aece-3bebe4242424.cloud.yugabyte.com:9042/yugabyte
+cassandra://<DB USER>:<DB PASSWORD>@4242424.cloud.yugabyte.com:9042/yugabyte
 ```
 
 Add the string to your application, replacing
 
 - `<DB USER>` with your database username.
 - `<DB PASSWORD>` with your database username.
```
Comment (pavithra-yb) [inline docs/content/latest/yugabyte-cloud/cloud-basics/connect-to-clusters.md:182]: <DB PASSWORD>` with your database password.

**Pull request merged**